get/apiserviceaccount.go

```diff
@@ -75,18 +75,12 @@ func (asa *apiServiceAccountsCmd) print(sas []iam.APIServiceAccount, get *Cmd, h
 }
 
 func (asa *apiServiceAccountsCmd) printToken(ctx context.Context, client *api.Client, sa *iam.APIServiceAccount) error {
-	secret, err := client.GetConnectionSecret(ctx, sa)
+	token, err := getConnectionSecret(ctx, client, "token", sa)
```

Comment:
I would still use the `tokenKey` const here.

```suggestion
	token, err := getConnectionSecret(ctx, client, tokenKey, sa)
```